app/index.js

```diff
@@ -21,7 +17,15 @@ async function appRender(whatToRender, whereToRender) {
 	return whereToRender;
 }
 
-function mouseClickHandler() {
+let reloaded = setInterval(() => {
+	loadWidget();
+}, 1800000);
```

Comment:
Create file with time constants

Reply:
Did you mean put function mouseClickHandler in other folder and then make export?